assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image-next-previous/settings.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import {
+	PanelBody,
+	__experimentalToggleGroupControl as ToggleGroupControl,
```

Comment:
I'd supress these warnings. I have found that we are already [doing this](https://github.com/woocommerce/woocommerce-blocks/blob/06de26e2e07e164015d52b6b8d815fa279262493/assets/js/atomic/blocks/product-elements/image/edit.tsx) in `image`.

```suggestion
	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
	// @ts-ignore - Ignoring because `__experimentalToggleGroupControl` is not yet in the type definitions.
	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
	__experimentalToggleGroupControl as ToggleGroupControl,
```